store.html

```diff
@@ -0,0 +1,191 @@
+                </li>
+                <li>
+                    <a href="store">Store</a>
+                </li>
+                <li>
+                    <a href="blog">Blog</a>
+                </li>
+                <li>
+                    <a href="help">Help</a>
+                </li>
+                <li>
+                    <a href="login">Login</a>
+                </li>
+    
+            </ul>
+        </nav>
+    
+    </header>
+<!--main-->
+    <main>
+        
+<!--form-->
+    <section class="form-container">
+         <form>
+            <div class="today-karma">
+            <p class="karma">Order your Karma device wifi today!</p>
+            </div>
+        <fieldset class="name">
+           <div class="groupname">
+            <label for="firstname">First Name <span class="requiredstar">*</span></label>
+            <input id="firstname" type="text" name="firstname" />
+            </div>
+            <div class="lastname">
+            <label for="lastame">Last Name <span class="requiredstar">*</span></label>
+            <input id="lastname" type="text" name="lastname" />
+         </div>
+    </fieldset>
+
+
+<!--address-->
+             <fieldset class="address-container">
+        <div class="address1">
+            <label for="adress">Address <span class="requiredstar">*</span></label>
+            <input id="address" type="text" name="address" />
+        </div>
+        <div class="address2">
+            <label for="adress">Address 2<span class="requiredstar">*</span></label>
+            <input id="address" type="text" name="address2" />
+        </div>
+    </fieldset>
+
+        <fieldset class="city-postcode"> 
+        <div class="city">
+            <label for="city">City <span class="requiredstar">*</span></label>
+            <select name="city" id="city" required>
+            
+            <option value="" >Select your city... </option>
+            <option value="brighton">Brighton</option> 
+            <option value="cambridge">Cambridge</option>
+            <option value="oxford">Oxford</option>
+            <option value="bristol">Bristol</option>
+            <option value="liverpool"> Liverpool</option>
+            <option value="manchester"> Manchester</option>
+            <option value="birmingham">Birmingham</option>
+            <option value="bournemouth">Bournemouth</option>
+         </select>
+        </div>
+        <div class="postcode">
+            <label for="postcode">Postcode<span class="requiredstar">*</span></label>
+            <input name="postcode" id="postcode" required>
+        </div>
+        </fieldset>
+<!--colour-->
+            
+                <div class="karmaorange">
+
+                    <div class="select-container">
+
+                    
+             <fieldset class="colour">
+                <legend>Select a colour</legend>
+                <input id="karmaorange" type="radio" name="colour" value="karmaorange" />
```

Comment:
The default option should be selected, so you can add 'checked' to the element:
```suggestion
                <input id="karmaorange" type="radio" name="colour" value="karmaorange" checked />
```